docs/guide/acquiring_data/fido.rst

```diff
@@ -180,131 +167,347 @@ client for results, in this case VSO.::
     <BLANKLINE>
     <BLANKLINE>
 
-To search for data from multiple instruments, wavelengths, times etc., use the
-pipe ``|`` operator. This joins queries together just as the logical ``OR``
-operator would::
+To search for data from multiple instruments, wavelengths, times etc., use the pipe ``|`` operator.
+This joins queries together just as the logical ``OR`` operator would::
 
     >>> Fido.search(a.Time('2012/3/4', '2012/3/4 02:00'),
     ...             a.Instrument.lyra | a.Instrument.rhessi)  # doctest: +REMOTE_DATA
     <sunpy.net.fido_factory.UnifiedResponse object at ...>
     Results from 3 Providers:
     <BLANKLINE>
     2 Results from the LYRAClient:
-         Start Time           End Time      Instrument ... Source Provider Level
-    ------------------- ------------------- ---------- ... ------ -------- -----
-    2012-03-04 00:00:00 2012-03-04 23:59:59       LYRA ... PROBA2      ESA     2
-    2012-03-04 00:00:00 2012-03-04 23:59:59       LYRA ... PROBA2      ESA     3
+           Start Time               End Time        Instrument ... Provider Level
+    ----------------------- ----------------------- ---------- ... -------- -----
+    2012-03-04 00:00:00.000 2012-03-04 23:59:59.999       LYRA ...      ESA     2
+    2012-03-04 00:00:00.000 2012-03-04 23:59:59.999       LYRA ...      ESA     3
     <BLANKLINE>
     1 Results from the RHESSIClient:
-         Start Time           End Time      Instrument ... Source Provider
-    ------------------- ------------------- ---------- ... ------ --------
-    2012-03-04 00:00:00 2012-03-04 23:59:59     RHESSI ... RHESSI     NASA
+           Start Time               End Time        Instrument ... Source Provider
+    ----------------------- ----------------------- ---------- ... ------ --------
+    2012-03-04 00:00:00.000 2012-03-04 23:59:59.999     RHESSI ... RHESSI     NASA
     <BLANKLINE>
     3 Results from the VSOClient:
-           Start Time               End Time        ... Extent Type      Info
-                                                    ...
-    ----------------------- ----------------------- ... ----------- --------------
-    2012-03-03 22:57:40.000 2012-03-04 00:33:20.000 ... PARTIAL_SUN RHESSI level-0
-    2012-03-04 00:33:20.000 2012-03-04 01:45:40.000 ... PARTIAL_SUN RHESSI level-0
-    2012-03-04 01:45:40.000 2012-03-04 02:09:00.000 ... PARTIAL_SUN RHESSI level-0
+           Start Time               End Time        ...   Size        Info
+                                                    ...  Mibyte
+    ----------------------- ----------------------- ... -------- --------------
+    2012-03-03 22:57:40.000 2012-03-04 00:33:20.000 ... -0.00098 RHESSI level-0
```

Comment:
Why is the size negative?

Reply:
because that's what VSO sent us... :shrug: